apps/omg/config/config.exs

```diff
@@ -13,4 +13,9 @@ config :omg, :eip_712_domain,
   version: "1",
   salt: "fad5c7f626d80f9256ef01929f3beb96e058b8b4b0e3fe52d84f054c0e2a7a83"
 
+config :omg, OMG.Utils.Tracer,
+  service: :omg,
+  adapter: SpandexDatadog.Adapter,
+  env: Atom.to_string(Mix.env())
```

Comment:
Should this be using the new [`APP_ENV`](https://github.com/omisego/devops/pull/103) value? Following up from a conversation with Ino, [MIX_ENV will be deprecated and set to prod](https://github.com/omisego/devops/pull/101#discussion_r294955427)

Same with the other configurations using `MIX.env()` in this PR.